### ISSUE TITLE
Branch event received after closed branch in storage throws exception

### ISSUE THREAD

**Issue by pmeijer** [webgme/webgme#1290]
https://sentry.io/vanderbilt-university/webgme/issues/203519783/

```
TypeError: Cannot read property 'getUpdateQueue' of undefined
  at _pullNextQueuedCommit(common/storage/storageclasses/editorstorage.js:650:0)
  at _remoteUpdateHandler(common/storage/storageclasses/editorstorage.js:261:0)
  at apply(common/storage/storageclasses/editorstorage.js:268:0)
  at apply(common/storage/socketio/websocket.js:34:0)
  at Socket.onack(/socket.io/socket.io.js:985:9)
  at Socket.onpacket(/socket.io/socket.io.js:909:12)
  at Manager.<anonymous>(/socket.io/socket.io.js:1512:15)
  at Manager.Emitter.emit(/socket.io/socket.io.js:1647:20)
  at Manager.ondecoded(/socket.io/socket.io.js:428:8)
  at Decoder.<anonymous>(/socket.io/socket.io.js:1512:15)
  at Decoder.Emitter.emit(/socket.io/socket.io.js:4389:20)
  at Decoder.add(/socket.io/socket.io.js:6740:12)
  at Manager.ondata(/socket.io/socket.io.js:418:16)
  at Socket.<anonymous>(/socket.io/socket.io.js:1512:15)
  at Socket.Emitter.emit(/socket.io/socket.io.js:4389:20)
  at Socket.onPacket(/socket.io/socket.io.js:2514:14)
  at WS.<anonymous>(/socket.io/socket.io.js:2331:10)
  at WS.Emitter.emit(/socket.io/socket.io.js:4389:20)
  at WS.Transport.onPacket(/socket.io/socket.io.js:2947:8)
  at WS.Transport.onData(/socket.io/socket.io.js:2939:8)
  at WebSocket.ws.onmessage(/socket.io/socket.io.js:4058:10)
```

**Comment by pmeijer** [comment]
https://sentry.io/vanderbilt-university/webgme/issues/203519783/

**Comment by pmeijer** [comment]
Closed by #1293
